certlogic/certlogic-js/src/evaluator.ts

```diff
@@ -75,7 +85,9 @@ const evaluateBinOp = (operator: string, values: CertLogicExpression[], data: an
         case "<":
         case ">":
         case "<=":
-        case ">=": {
+        case ">=":
+        case "before":
+        case "not-after": {
```

Comment:
Since operators `after` and `not-before` are already mentioned above. Maybe makes sense immediately to list them here

Reply:
In fact: *must* be listed here - was an omission. I don't test the runtime exception cases, which is why I didn't catch this.